extensions/core/runtime/src/main/java/org/apache/camel/quarkus/core/CamelServiceFilter.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.quarkus.core;
+
+import java.util.function.Predicate;
+
+@FunctionalInterface
+public interface CamelServiceFilter extends Predicate<CamelServiceInfo> {
```

Comment:
I fail to see what value is CamelServiceFilter interface adding. I'd personally prefer using `Predicate<CamelServiceInfo>` directly because that way I'd immediately see that it is just a Predicate and nothing else.